docs/advanced-topics/docker-alternatives.rst

```diff
@@ -40,10 +40,10 @@ More information about compatibility of dockerfiles with Singularity
 can be found `here <https://sylabs.io/guides/3.4/user-guide/singularity_and_docker.html#best-practices>`__.
 
 Singularity can be installed following the instructions
-`here <https://sylabs.io/guides/3.4/user-guide/quick_start.html>`__. Note that the installation is relatively complicated
-and requires ``sudo``. Neither the macOS version (Singularity Desktop) nor the Debian/Ubuntu package version currently
-available (2.6.1) is compatible. You will need to download a version >3.0.0 and build it from source.
+`here <https://github.com/sylabs/singularity/blob/main/INSTALL.md>`__.
 
+.. note:: Multiple installations of Go has been seen to cause a ``Go compiler not found`` error when installing Singularity.
```

Comment:
Consider replacing "has been seen" with "has been known"

Reply:
Whichever way you go, I just noticed, it should be "have", not "has", to agree with "Multiple installations".